template_project/app/src/main/AndroidManifest.xml

```diff
@@ -33,9 +33,11 @@
             android:name="asset_statements"
             android:resource="@string/assetStatements" />
 
-        <meta-data
-            android:name="cros_web_alternative"
-            android:value="@string/crosLaunchUrl" />
+        <% if (webManifestUrl) { %>
```

Comment:
Huh, I did not know we could use substitutions in the AndroidManifest.xml

Reply:
That's lodash, the template engine we're using in llama-pack.